lighthouse-core/test/gather/driver-test.js

```diff
@@ -46,9 +48,23 @@ function createActiveWorker(id, url, controlledClients, status = 'activated') {
   };
 }
 
+function createOnceMethodResponse(method, response) {
```

Comment:
someday using jests mock functions would be awesome :)

Reply:
yeah our lame driver/connection mocking everywhere is ripe for a cleanup.